code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -98,6 +98,18 @@ emp_act
 			visible_message("<span class='danger'>[src] blocks [attack_text] with [r_hand]!</span>", \
 							"<span class='userdanger'>[src] blocks [attack_text] with [r_hand]!</span>")
 			return 1
+	if(l_hand && istype(l_hand, /obj/item/weapon))//Ling-shields 100% block attacks, but they break after (Number of genomes) x 2
+		var/obj/item/weapon/I = l_hand
+		if(istype(I, /obj/item/weapon/shield/changeling))

```

Comment:
congratulations, you see the reasoning for the existence of line comments and the review process!